GVRf/Extensions/3DCursor/IODevices/gvrmobileiodevice/src/main/java/com/example/gvrmobileiodevice/ClientWrite.java

```diff
@@ -0,0 +1,36 @@
+package com.example.gvrmobileiodevice;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+
+/**
+ * Created by v.nidamarthy on 9/2/16.
+ *
+ * Class to write the one byte data to send haptic feedback for vibration
+ */
+public class ClientWrite implements Runnable
+{
+    GVRMobileIoDevice.ServerInfo serverInfo = null;
+    ClientWrite(GVRMobileIoDevice.ServerInfo serverinfo)
+    {
+        serverInfo = serverinfo;
+    }
+    @Override
+    public void run() {
+        try {
+                byte[] bytes = new byte[1];
+                bytes[0] = (byte)1;
```

Comment:
what is this used for. You can use a constant instead of a hardcoded value, this makes more readable and easier to extend to multiple messages in the future.